tdvt/tdvt/config_gen/datasource_list.py

```diff
@@ -74,6 +74,17 @@ def get_expected_message(config):
     return config.get('ExpectedMessage', '')
 
 
+def get_is_smoke_test(config):
+    return config.get('SmokeTest', 'False') == 'True'
```

Comment:
I'm kind of worried about using string compare vs bools. But I see why it's done.

Reply:
I agree it's ugly. I could change the setting to look for `yes` and `no`, which at least would avoid the appearance of stringy bools.

Reply:
I feel that it might be better to change the def name to def is_smoke_test(config).

Reply:
That's a good point, @sonia0504. I named it that way to keep the names similar to `get_password_file` and `get_expected_msg`. But I'm happy to change it.